examples/basic.rs

```diff
@@ -92,7 +92,7 @@ fn main() {
         siv.add_layer(
             Dialog::around(TextView::new(format!("{} / {:?}", column.as_str(), order)))
                    .title("Sorted by")
-                   .button("Close", |s| s.pop_layer())
+                   .button("Close", |s| drop(s.pop_layer())) // drop to avoid compile-time error 
```

Comment:
You can also ignore the result: `|s| { s.pop_layer(); }`

Reply:
Would you prefer it that way? Easy enough to give you a PR that looks that way, rather than this way.